synchronizer/synchronizer.go

```diff
@@ -36,6 +36,9 @@ type Synchronizer struct {
 	duration ViewDuration
 	timer    *time.Timer
 
+	// number of cuncurrent views, 0 and 1 both give no concurrency.
+	pipelinedViews uint8
```

Comment:
You could make this be of the type `hotstuff.View`; then you wouldn't need to cast it all the time, and could add it directly to other views. But maybe it would be confusing?

Reply:
done in [eb95aa1](https://github.com/relab/hotstuff/pull/78/commits/eb95aa1b566321e76dec2296f05e3403fe58758e) but see comment above.

Reply:
I changed it to `uint32` in [df3fc32](https://github.com/relab/hotstuff/pull/78/commits/df3fc324538afeeadc39d8adedcddb7d618f4ee8). 
This avoids a cast, since it is transferred by protobuf as `uint32`
```proto
 // Number of blocks proposed concurrently
  uint32 PipelinedViews = 22;
```